<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify_func</class>
 <widget class="QWidget" name="Modify_func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>687</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>687</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查看</string>
  </property>
  <widget class="QLabel" name="image_lable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="photo_pb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改头像</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#plainTextEdit
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>17</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_le">
      <property name="styleSheet">
       <string notr="true">#name_le
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>电话号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_le">
      <property name="styleSheet">
       <string notr="true">#phone_le
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生日     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">#dateEdit
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_le">
      <property name="styleSheet">
       <string notr="true">#email_le
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分组      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="type_le">
      <property name="styleSheet">
       <string notr="true">#type_le
{
background-color: rgba(255, 255, 255,0.8);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 14pt; /* 文本字体大小 */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
